Lib/test/test_select.py

```diff
@@ -87,6 +87,10 @@ def fileno(self):
         a[:] = [F()] * 10
         self.assertEqual(select.select([], a, []), ([], a[:5], []))
 
+    def test_disallow_instantiation(self):
+        tp = type(select.poll())
+        self.assertRaises(TypeError, tp)
```

Comment:
Later we might use an helper function to also check the error message.

Reply:
Let's add it in a separate PR. I'll update and rebase onto `main` when the helper has landed.

Reply:
Sure, I'm fine with with `self.assertRaises(TypeError)` for now. I mostly care about fixing https://bugs.python.org/issue43916 release blocker. Once it's no longer a release blocker, we can enhance the implementation.